docs/en/query_language/create.md

```diff
@@ -299,4 +299,149 @@ Depending on dictionary [layout](dicts/external_dicts_dict_layout.md) one or mor
 
 For more information, see [External Dictionaries](dicts/external_dicts.md) section.
 
+## CREATE USER {#create-user-statement}
+
+Creates a user account.
+
+ClickHouse allows to configure user accounts in the [user.xml](../operations/settings/settings_users.md) file. The `CREATE USER` statement is an alternative way of creating user accounts.
```

Comment:
We'd better make it look like CREATE USER is a main method, while xml file is a legacy alternative.
Also needs some explanation on what happens if there are conflicts between two methods.